<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admineditableinv</class>
 <widget class="QWidget" name="admineditableinv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,
                            stop:0 #e3f2fd, stop:1 #bbdefb);
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>-3</y>
     <width>951</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget_products">
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
QTableWidget {
    background-color: #E6F2FF; /* Light blue for even rows */
alternate-background-color: #B5D6F5;

}

QHeaderView::section {
    background-color: #3498DB;  /* Optional: blue header */
    color: white;
    font-weight: bold;
}
</string>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Product_ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Unit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Barcode</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>500</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QPushButton" name="pushButton_updateproduct">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #28a745;  /* Bootstrap Green */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}
QPushButton:hover {
    background-color: #218838;  /* Darker green on hover */
}
</string>
            </property>
            <property name="text">
             <string>Update Product</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_delete">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #E74C3C;  /* Bright red */
    color: white;
 font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #C0392B;  /* Darker red on hover */
}
</string>
            </property>
            <property name="text">
             <string>Delete Product</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">
QLabel {
background-color: #3498DB;  /* Medium gray */
color: white;

    border-radius: 5px;
    padding:  10px;
font-size: 10px;
    font-weight: bold;
}
</string>
                </property>
                <property name="text">
                 <string>Search Product:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_searchproduct">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    background-color: #ECF0F1;
    color: #2C3E50;
    border: 1px solid #BDC3C7;
    border-radius: 3px;
    padding: 2px 4px;
    font-size: 12px;
    min-height: 20px; /* reduce height */
}

QLineEdit:focus {
    border: 1px solid #3498DB;
    background-color: #FFFFFF;
}

QLineEdit::placeholder {
    color: #95A5A6;
    font-style: italic;
}
</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_refresh">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #28a745;  /* Bootstrap Green */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}
QPushButton:hover {
    background-color: #218838;  /* Darker green on hover */
}
</string>
              </property>
              <property name="text">
               <string>Refresh</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">QLabel {
background-color:#5D6D7E;  /* Medium gray */
color: white;

    border-radius: 5px;
    padding: 5px 10px;
font-size: 12px;
    font-weight: bold;
}
</string>
            </property>
            <property name="text">
             <string>ADD Product by Filling Form:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>400</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">QLabel {
background-color: #E67E22;  /* Medium gray */
color: white;
 font-weight: bold;
font-size: 10px;
    border-radius: 5px;
    padding: 5px 10px;
}
</string>
              </property>
              <property name="text">
               <string>Product Id:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_productid">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #ECF0F1;
    color: #2C3E50;
    border: 1px solid #BDC3C7;
    border-radius: 3px;
    padding: 2px 4px;
    font-size: 12px;
    min-height: 20px; /* reduce height */
}

QLineEdit:focus {
    border: 1px solid #3498DB;
    background-color: #FFFFFF;
}

QLineEdit::placeholder {
    color: #95A5A6;
    font-style: italic;
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">QLabel {
background-color: #E67E22;  /* Medium gray */
color: white;
 font-weight: bold;
font-size: 10px;
    border-radius: 5px;
    padding: 5px 10px;
}
</string>
              </property>
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_productname">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #ECF0F1;
    color: #2C3E50;
    border: 1px solid #BDC3C7;
    border-radius: 3px;
    padding: 2px 4px;
    font-size: 12px;
    min-height: 20px; /* reduce height */
}

QLineEdit:focus {
    border: 1px solid #3498DB;
    background-color: #FFFFFF;
}

QLineEdit::placeholder {
    color: #95A5A6;
    font-style: italic;
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="styleSheet">
                 <string notr="true">QLabel {
background-color: #E67E22;  /* Medium gray */
color: white;
 font-weight: bold;
font-size: 10px;
    border-radius: 5px;
    padding: 5px 10px;
}
</string>
                </property>
                <property name="text">
                 <string>Category:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_category">
                <property name="styleSheet">
                 <string notr="true">QComboBox {
    background-color: #f0f0f0;         /* Light gray background */
    border: 1px solid #ccc;            /* Soft border */
    border-radius: 5px;
    padding: 5px 10px;
    font-weight: bold;
    color: #333;                       /* Dark text */
}

QComboBox:hover {
    background-color: #e2e6ea;         /* Slightly darker on hover */
    border: 1px solid #888;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #ccc;
}

QComboBox::down-arrow {
    image: url(:/icons/arrow-down.png);  /* Optional: Add a custom arrow icon */
}
</string>
                </property>
                <item>
                 <property name="text">
                  <string>Bevarage</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Grocery</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dairy</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Fruit</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Vegetable</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Cosmetics</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Miscellaneous</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">QLabel {
background-color: #E67E22;  /* Medium gray */
color: white;
 font-weight: bold;
font-size: 10px;
    border-radius: 5px;
    padding: 5px 10px;
}
</string>
                </property>
                <property name="text">
                 <string>Quantity:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_productquantity">
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    background-color: #ECF0F1;
    color: #2C3E50;
    border: 1px solid #BDC3C7;
    border-radius: 3px;
    padding: 2px 4px;
    font-size: 12px;
    min-height: 20px; /* reduce height */
}

QLineEdit:focus {
    border: 1px solid #3498DB;
    background-color: #FFFFFF;
}

QLineEdit::placeholder {
    color: #95A5A6;
    font-style: italic;
}
</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="styleSheet">
                   <string notr="true">QLabel {
background-color: #E67E22;  /* Medium gray */
color: white;
 font-weight: bold;
font-size: 10px;
    border-radius: 5px;
    padding: 5px 10px;
}
</string>
                  </property>
                  <property name="text">
                   <string>Price:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_productprice">
                  <property name="styleSheet">
                   <string notr="true">QLineEdit {
    background-color: #ECF0F1;
    color: #2C3E50;
    border: 1px solid #BDC3C7;
    border-radius: 3px;
    padding: 2px 4px;
    font-size: 12px;
    min-height: 20px; /* reduce height */
}

QLineEdit:focus {
    border: 1px solid #3498DB;
    background-color: #FFFFFF;
}

QLineEdit::placeholder {
    color: #95A5A6;
    font-style: italic;
}
</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="styleSheet">
                 <string notr="true">QLabel {
background-color: #E67E22;  /* Medium gray */
color: white;
 font-weight: bold;
font-size: 10px;
    border-radius: 5px;
    padding: 5px 10px;
}
</string>
                </property>
                <property name="text">
                 <string>Size:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_size">
                <property name="styleSheet">
                 <string notr="true">QComboBox {
    background-color: #f0f0f0;         /* Light gray background */
    border: 1px solid #ccc;            /* Soft border */
    border-radius: 5px;
    padding: 5px 10px;
    font-weight: bold;
    color: #333;                       /* Dark text */
}

QComboBox:hover {
    background-color: #e2e6ea;         /* Slightly darker on hover */
    border: 1px solid #888;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #ccc;
}

QComboBox::down-arrow {
    image: url(:/icons/arrow-down.png);  /* Optional: Add a custom arrow icon */
}
</string>
                </property>
                <item>
                 <property name="text">
                  <string>N/A</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>XS</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>S</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>M</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>L</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>XL</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>XXL</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>XXXL</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">QLabel {
background-color: #E67E22;  /* Medium gray */
color: white;
 font-weight: bold;
font-size: 10px;
    border-radius: 5px;
    padding: 5px 10px;
}
</string>
              </property>
              <property name="text">
               <string>Unit:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_unit">
              <property name="styleSheet">
               <string notr="true">QComboBox {
    background-color: #f0f0f0;         /* Light gray background */
    border: 1px solid #ccc;            /* Soft border */
    border-radius: 5px;
    padding: 5px 10px;
    font-weight: bold;
    color: #333;                       /* Dark text */
}

QComboBox:hover {
    background-color: #e2e6ea;         /* Slightly darker on hover */
    border: 1px solid #888;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #ccc;
}

QComboBox::down-arrow {
    image: url(:/icons/arrow-down.png);  /* Optional: Add a custom arrow icon */
}
</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>Kg</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>pcs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Liter</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Miscellaneous</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QComboBox" name="comboBox_barcodeoption">
            <item>
             <property name="text">
              <string>Scan Barcode</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Generate Barcode</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_barcode">
            <property name="styleSheet">
             <string notr="true">QLabel {
    background-color: #FFF9DB;  /* Light yellow */
    color: #2C3E50;             /* Dark gray text for contrast */
    font-weight: bold;
    font-size: 10px;
    border-radius: 5px;
    padding: 5px 10px;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>400</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_add">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #28a745;  /* Bootstrap Green */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}
QPushButton:hover {
    background-color: #218838;  /* Darker green on hover */
}
</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
